src/models/db/migrations/20240628090906_query_table.ts

```diff
@@ -0,0 +1,39 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  const now = () => knex.fn.now()
+
+  await knex.schema.createTable('queries', (def) => {
+    def.uuid('id').defaultTo(knex.raw('uuid_generate_v4()')).primary()
+    def.uuid('connection_id').notNullable()
+    def.string('company_name').notNullable()
+    def.string('query_type').notNullable()
+    def
+      .enum('direction', ['Sent', 'Received'], {
+        useNative: true,
+        enumName: 'query_direction',
+      })
+      .notNullable()
+    def
+      .enum('status', ['Resolved', 'Pending Your Input', 'Pending Their Input'], {
+        useNative: true,
+        enumName: 'query_status',
+      })
+      .notNullable()
```

Comment:
I'd prefer enum values to be lower snake case and then do a transform in the typescript.